agent/wptdriver/wpt_test.cc

```diff
@@ -286,8 +288,13 @@ bool WptTest::Load(CString& test) {
         else if (!key.CompareNoCase(_T("addCmdLine")))
           _browser_additional_command_line = value;
         else if (!key.CompareNoCase(_T("continuousVideo")) &&
-                 _ttoi(value.Trim()))
+          _ttoi(value.Trim()))
           _continuous_video = true;
+        else if (!key.CompareNoCase(_T("timeout"))) {
+          _overall_timeout_in_ms = _ttoi(value.Trim()) * 1000;

```

Comment:
_test_timeout is actually used for a per-step basis, and is reset after every step. Since we want to keep track of the overall test time(including time spent between steps), I created a new field.